neuralmonkey/evaluators/f1_bio.py

```diff
@@ -45,13 +40,13 @@ def chunk2set(seq):
         return output
 
     @staticmethod
-    def f1(decoded, reference):
+    def f1_score(decoded: List[str], reference: List[str]) -> float:
         set_dec = F1Evaluator.chunk2set(decoded)
         set_ref = F1Evaluator.chunk2set(reference)
 
-        tp = float(len(set_dec.intersection(set_ref)))
-        if tp == 0.0:
+        true_positives = float(len(set_dec.intersection(set_ref)))
+        if true_positives == 0.0:
             return 0.0
-        precision = tp / len(set_dec)
-        recall = tp / len(set_ref)
+        precision = true_positives / len(set_dec)
+        recall = true_positives / len(set_ref)
         return 2.0 * precision * recall / (precision + recall)
```

Comment:
Jednoho dne jo. Ale udělal bych to, až se odhodláme udělat pořádně ty `__init__.py` soubory.